public/app/features/alerting/unified/AlertEditor.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { FC } from 'react';
```

Comment:
this is probably **Rule**Editor... "Alert" is very ambiguos :)

Reply:
I agree. Alert is overloaded. i'm glad we'll have some standardization on terms here soon 😄